ci_scripts/ci-post-clone.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
+
+#  ci-post-clone.sh
+#  Multisig
+#
+#  Created by Dirk Jäckel on 17.07.23.
+#  Copyright © 2023 Gnosis Ltd. All rights reserved.
+
+env
+echo `pwd`
```

Comment:
where and how is ci-post-clone.sh is called? Could you please add some description comments. Also explanation why it is needed to move up a directory.

Reply:
See: https://developer.apple.com/documentation/xcode/writing-custom-build-scripts
I used `cd ..` because I do not trust our scripts to work from anywhere.

Reply:
Comments added